### DISCUSSION TITLE
Exercise 22: Paxos live with a failure detector in W.

### DISCUSSION THREAD

**Discussion by dag7dev** [sapienzastudentsnetwork/distributed-systems#47] [Exercises]
**Exercise 22:** Show that you can make Paxos live with a failure detector in $W$.

**Comment by Tommaso-Sgroi** [comment]
By definition a weak failure detector has weak accuracy and weak completeness. Weak accuracy means that there exists at least one process $p$ which is never suspected by any correct process. That means that the intersection of suspected correct processes set is never void; so we can exchange the correct set with each other and elect as coordinator the smallest node id in the intersection. So we will make Paxos live with ONLY a weak failure detector.
